nordicsemi/dfu/bl_dfu_sett.py

```diff
@@ -167,17 +167,18 @@ def generate(self, arch, app_file, app_ver, bl_ver, bl_sett_ver, custom_bl_sett_
             # calculate application size and CRC32
             self.app_sz = int(Package.calculate_file_size(self.app_bin)) & 0xffffffff
             self.app_crc = int(Package.calculate_crc(32, self.app_bin)) & 0xffffffff
+            self.bank0_bank_code = 0x1 & 0xffffffff
         else:
             self.app_sz = 0x0 & 0xffffffff
             self.app_crc = 0x0 & 0xffffffff
+            self.bank0_bank_code = 0x0 & 0xffffffff
```

Comment:
This should be a 32 bit zero (I'm not intimately familiar with python uints and bitmasks, so maybe this happens already)

    self.bank0_bank_code = 0x0

or strictly speaking

    self.bank0_bank_code = 0x00000000

if there is any difference.

Reply:
I don't really understand your question here. In my understanding, variables are bitmasked with '0xffffffff' to make sure they fit on 32 bits to make sure that struct.pack() doesn't throw an error.
In case of hard-coded value such as zero, I think the bitmasking operation is useless, but as it was done elsewhere, I kept that style.

Reply:
Ok, I see. As long as it ends up as a 4 byte value where all 4 bytes are 0, that's fine.